Automattic/jetpack
# Boost: Add Async Options

#### Changes proposed in this Pull Request:
This provides a streamlined way to talk to WordPress from the UI and handling nonces, REST API calls and reactive option stores.

#### Other information:


#### Jetpack product discussion
No link - we've discussed this in our team calls. 

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Apply the patch
2. Import async `options.ts` in Svelte and make sure that you can `console.log( API )`
